api/constants.js

```diff
@@ -0,0 +1,11 @@
+export const CsvRequest = {
+  NO_REQUEST: 'NO_REQUEST',
+  FULL_REPORT: 'FULL_REPORT'
+}
+
+export const CsvRequestStatus = {
+  NO_REQUEST: 'NO_REQUEST',
+  FULL_REPORT: 'FULL_REPORT',

```

Comment:
Why is `FULL_REPORT` used here? Wouldn't be `DONE` or `COMPLETED` be more intuitive?

Reply:
Another case of me thinking more complicated than necessary... to distinguish / match the completion status to potentially different kinds of requests in the future. Nothing for this PR...I've now changed it to DONE (and I shortened GENERATING_REPORT to a simpler IN_PROGRESS).